fe/fe-core/src/main/java/com/starrocks/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -517,17 +525,84 @@ private void setCustomKafkaProperties(Map<String, String> kafkaProperties) {
 
     private void setDefaultKafkaGroupID() {
         if (this.customProperties.containsKey(PROPERTY_KAFKA_GROUP_ID)) {
+            useDefaultGroupId = false;
             return;
         }
         this.customProperties.put(PROPERTY_KAFKA_GROUP_ID, name + "_" + UUID.randomUUID());
     }
 
-    @VisibleForTesting
     @Override
-    protected String dataSourcePropertiesJsonToString() {
-        Map<String, String> dataSourceProperties = Maps.newHashMap();
-        dataSourceProperties.put("brokerList", brokerList);
-        dataSourceProperties.put("topic", topic);
+    public String dataSourcePropertiesToSql() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("(\n");
+        sb.append("\"").append(CreateRoutineLoadStmt.KAFKA_BROKER_LIST_PROPERTY).append("\"=\"");
+        sb.append(brokerList).append("\",\n");
```

Comment:
Is it necessarry to add `\n` here?

Reply:
> Is it necessarry to add `\n` here?

Yes, each attribute should be on a new line.